recipes/gridded/meta.yaml

```diff
@@ -0,0 +1,67 @@
+# Note: there are many handy hints in comments in this example -- remove them when you've finalized your recipe
+
+# Jinja variables help maintain the recipe as you'll update the version only here.
+{% set name = "gridded" %}
+{% set version = "0.0.7" %}
+{% set sha256 = "679463336d74db18c85f9cf8935baedce4e088a3b8d20a5f1dafb76afb07864e" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://github.com/NOAA-ORR-ERD/gridded/archive/v{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+    number: 2
+    script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - six
+  run:
+    - python
+    - six
+    - numpy
+    - scipy
+    - netcdf4
+    - cell_tree2d
+
+test:
+  requires:
```

Comment:
Dependencies that are part of the `run` requirements shouldn't be repeated in the `test/requires` section.